src/framework/components/collision/component.js

```diff
@@ -229,55 +262,76 @@ class CollisionComponent extends Component {
         this._setValue('axis', arg);
     }
 
+    /**
+     * Gets the local space axis with which the capsule, cylinder or cone-shaped collision volume's
+     * length is aligned.
+     *
+     * @type {number}
+     */
     get axis() {
         return this.data.axis;
     }
 
     /**
-     * The total height of the capsule, cylinder or cone-shaped collision  volume from tip to tip.
-     * Defaults to 2.
+     * Sets the total height of the capsule, cylinder or cone-shaped collision volume from tip to
+     * tip. Defaults to 2.
      *
      * @type {number}
      */
     set height(arg) {
         this._setValue('height', arg);
     }
 
+    /**
+     * Gets the total height of the capsule, cylinder or cone-shaped collision volume from tip to
+     * tip.
+     *
+     * @type {number}
+     */
     get height() {
         return this.data.height;
     }
 
     /**
-     * The asset for the model of the mesh collision volume - can also  be an asset id. Defaults to
-     * null.
+     * Sets the asset or asset id for the model of the mesh collision volume. Defaults to null.
      *
-     * @type {Asset}
+     * @type {Asset|number|null}
      */
     set asset(arg) {
         this._setValue('asset', arg);
     }
 
+    /**
+     * Gets the asset or asset id for the model of the mesh collision volume.
+     *
+     * @type {Asset|number|null}
+     */
     get asset() {
         return this.data.asset;
     }
 
     /**
-     * The render asset of the mesh collision volume - can also be  an asset id. Defaults to null.
+     * Sets the render asset or asset id of the mesh collision volume. Defaults to null.
```

Comment:
"Defaults to null" doesn't seem right on a setter. Should be on a getter probably.